controllers/producersController.js

```diff
@@ -1,85 +1,88 @@
 const Producer = require('../models/producer');
+const { ADD_PRODUCER, DELETE_PRODUCER, UPDATE_PRODUCER } = require('./constants');
 
 module.exports = {
-  producerCreate: async(req, res) => {
+    producerCreate: async(req, res) => {
     //  post new producer to the database
-    let producer = new Producer();
+        //  body parser lets us use the req.body
+        const {
+            firstName,
+            lastName,
+            phoneNumber,
+            country,
+            state,
+            localGovernment,
+            address,
+            typeOfProducts,
+            createdAt,
+            updatedAt,
+            deletedAt
+        } = req.body;
 
-    //  body parser lets us use the req.body
-    const {
-      firstName,
-      lastName,
-      phoneNumber,
-      country,
-      state,
-      localGovernment,
-      address,
-      typeOfProducts,
-      createdAt,
-      updatedAt,
-      deletedAt
-    } = req.body;
+        const producer = Object
+            .assign(new Producer, {
```

Comment:
`new Producer` is not javascript syntax

Reply:
whats the proper js syntax here.
it should be ```new Producer()```